index.bs

```diff
@@ -318,6 +339,285 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
+
+<table class="data">
+  <caption style="caption-side:bottom">
+    Table 2 - Meaning of the value of <code>token</code>.
+  </caption>
+  <thead>
+    <tr>
+      <th>Value of <code>token</code></th>
+      <th>Name</th>
+      <th>Input count</th>
+      <th>Meaning before pushing to the stack</th>
+      <th>Value pushed to the stack<br>(<math><mi>L</mi></math> and <math><mi>R</mi></math> being operands popped from the stack for operators)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>0</td>
+      <td>constant operand</td>
+      <td>0</td>
+      <td><math><msup><mn>2</mn><mrow><mi>bit_depth</mi><mo>+</mo><mn>3</mn></mrow></math> bits from the stream read as a signed integer.</td>
+      <td>constant value</td>
+    </tr>
+    <tr>
+      <td>1</td>
+      <td>sample operand</td>
+      <td>0</td>
+      <td>Sample value from the <code>input_image_item_index</code><sup>th</sup> input image item.</td>
+      <td>input image item sample value</td>
+    </tr>
+    <tr>
+      <td>2</td>
+      <td>negate operator</td>
+      <td>1</td>
+      <td>Negation of the left operand.</td>
+      <td><math><mo>-</mo><mi>L</mi></math></td>
+    </tr>
+    <tr>
+      <td>3</td>
+      <td>absolute value operator</td>
+      <td>1</td>
+      <td>Absolute value of the left operand.</td>
+      <td><math><mo>|</mo><mi>L</mi><mo>|</mo></math></td>
+    </tr>
+    <tr>
+      <td>4</td>
+      <td>not operator</td>
+      <td>1</td>
+      <td>Bitwise complement of the operand.</td>
+      <td><math><mo>¬</mo><mi>L</mi></math></td>
+    </tr>
+    <tr>
+      <td>5</td>
+      <td>bsr operator</td>
+      <td>1</td>
+      <td>0-based index of the most significant set bit of the left operand if the left operand is strictly positive, zero otherwise.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mn>0</mn></mtd><mtd><mo>if</mo><mi>L</mi><mo>≤</mo><mn>0</mn></mtd></mtr>
+            <mtr><mtd>truncate<mo>(</mo><msub><mo>log</mo><mn>2</mn></msub><mi>L</mi><mo>)</mo></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>128</td>
+      <td>sum operator</td>
+      <td>2</td>
+      <td>Left operand added to the right operand.</td>
+      <td><math><mi>L</mi><mo>+</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>129</td>
+      <td>difference operator</td>
+      <td>2</td>
+      <td>Right operand subtracted from the left operand.</td>
+      <td><math><mi>L</mi><mo>-</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>130</td>
+      <td>product operator</td>
+      <td>2</td>
+      <td>Left operand multiplied by the right operand.</td>
+      <td><math><mi>L</mi><mo>×</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>131</td>
+      <td>divide operator</td>
```

Comment:
Nit: The names of the four common arithmetic operators have inconsistent styles. I suggest either of the following.

1. Name them by the output of the operator: sum, difference, product, **quotient**
2. Name them by the action (a verb) of the operator: **add**, **subtract**, **multiply**, divide